assets/src/scss/components/_avatars.scss

```diff
@@ -137,51 +137,27 @@
         }
     }
     .avatar-intial{
-        background-color: $border-color;
         border-radius: 50%;
         padding: 11px 18px;
-        &:nth-child(3){
-            padding: 10px 13px;
-        }
-        &:nth-child(4){
-            padding: 9px 17px;
-        }
-        &:nth-child(5){
-            padding: 8px 13px;
-        }
-        &:last-child{
-            padding: 6px 11px;
-        }
-        h1,h2,h3,h4{
-            font-weight: 500;
-        }
     }
     .ping{
-        background: #fff;
-        border-radius: 50%;
-        border: 3px solid #222;
-        width: 0px;
-        height: 0px;
         animation: load 1.5s ease-out infinite;
-        position: absolute;
-        top: 9px;
-        right: 13px;
     }
     @keyframes load {
         0% {
-            background: transparent;
-            border: 3px solid transparent;
+            background-color: rgb(54, 155, 54);
```

Comment:
use css variable only.not use color directly

Reply:
![Screenshot from 2023-02-28 13-49-07](https://user-images.githubusercontent.com/108167253/221794332-90bea000-c0ca-41c5-9157-dbfb8f77f50f.png)